src/Symfony/Component/Security/CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 CHANGELOG
 =========
 
+4.4.0
+-----
+
+ * Added `ChainPasswordEncoder`
+ * Added method `PasswordEncoderInterface::needsRehash()`
+ * Added and implemented `PasswordUpgraderInterface`
+ * Deprecated `BCryptPasswordEncoder`, use `NativePasswordEncoder` instead
```

Comment:
this last line looks wrong. Deprecating it is already done in 4.3 AFAICT

Reply:
fixed